packages/@react-aria/radio/src/useRadioGroup.ts

```diff
@@ -130,6 +134,8 @@ export function useRadioGroup(props: AriaRadioGroupProps, state: RadioGroupState
       ...fieldProps,
       ...focusWithinProps
     }),
-    labelProps
+    labelProps,
+    descriptionProps,
+    errorMessageProps
   };
```

Comment:
Noticed that VO on Mac at least does not announce the help text or error message when navigating to a radio button. We might need to add the aria-describedby to each individual radio as well? Same issue with checkbox group.

For the error message I think it makes sense to put it on every checkbox/radio. But for description, maybe just the first/last so it's not too verbose? Might need to do some testing...

Reply:
I've updated it to include aria-describedby on individual checkboxes and radio items. IMO after testing, having the error message on every item didn't feel excessive because there's a pause before it's read. I'll let people test it themselves, and I can change it to read the error on first and last if that's preferred.